Shared/Samples/Get elevation at point on surface/README.md

```diff
@@ -0,0 +1,31 @@
+# Get elevation at point on surface
+
+Get the elevation for a given point on a surface in a scene.
+
+![Screenshot of get elevation at point on surface sample](get-elevation-at-point-on-surface.png)
+
+## Use case
+
+Knowing the elevation at a given point in a landscape can aid in navigation, planning and survey in the field.
+
+## How to use the sample
+
+Tap anywhere on the surface to get the elevation at that point. Elevation is reported in the standard local measurements. The scene view is in WGS84, which is a geographic coordinate system.
```

Comment:
https://support.esri.com/en-us/gis-dictionary/wgs84 WGS 84 or WGS84 seem to be both acceptable proper nouns.